ci/jenkins/jobs/projects.yaml

```diff
@@ -562,6 +562,196 @@
+          status_context: nephe-test-e2e-with-windows-eks
+          status_url: ""
+          success_status: Build finished.
+          failure_status: Failed. Add comment /nephe-test-e2e-with-windows-eks to re-trigger.
+          error_status: Failed. Add comment /nephe-test-e2e-with-windows-eks to re-trigger.
+          triggered_status: null
+          started_status: null
+          concurrent: true
+          wrappers:
+            - credentials-binding:
+                - text:
+                    credential-id: CODECOV_TOKEN # Jenkins secret that stores codecov token
+                    variable: CODECOV_TOKEN
+            - timeout:
+                fail: true
+                timeout: 120
+                type: absolute
+            - credentials-binding:
+                - text:
+                    credential-id: VMC_NETWORK_0
+                    variable: VMC_NETWORK_0
+                - text:
+                    credential-id: VMC_NETWORK_1
+                    variable: VMC_NETWORK_1
+                - text:
+                    credential-id: VCENTERNAME
+                    variable: VCENTERNAME
+                - text:
+                    credential-id: DATACENTERNAME
+                    variable: DATACENTERNAME
+                - text:
+                    credential-id: RESOURCEPOOLPATH
+                    variable: RESOURCEPOOLPATH
+                - text:
+                    credential-id: VC_USER
+                    variable: VC_USER
+                - text:
+                    credential-id: GOVC_PASSWORD
+                    variable: GOVC_PASSWORD
+                - text:
+                    credential-id: VC_HOST
+                    variable: VC_HOST
+                - text:
+                    credential-id: DATA_STORE
+                    variable: DATA_STORE
+                - text:
+                    credential-id: VC_CLUSTER
+                    variable: VC_CLUSTER
+                - text:
+                    credential-id: VIRTUAL_MACHINE
+                    variable: VIRTUAL_MACHINE
+                - text:
+                    credential-id: ANTREA_DOCKER_USERNAME
+                    variable: ANTREA_DOCKER_USERNAME
+                - text:
+                    credential-id: ANTREA_DOCKER_PWD
+                    variable: ANTREA_DOCKER_PWD
+                - text:
+                    credential-id: AWS_ACCESS_KEY_ID
+                    variable: AWS_ACCESS_KEY_ID
+                - text:
+                    credential-id: AWS_ACCESS_KEY_SECRET
+                    variable: AWS_ACCESS_KEY_SECRET
+                - text:
+                    credential-id: AWS_KEY_PAIR_NAME
+                    variable: AWS_KEY_PAIR_NAME
+                - text:
+                    credential-id: EKS_IAM_ROLE
+                    variable: EKS_IAM_ROLE
+                - text:
+                    credential-id: EKS_IAM_INSTANCE_PROFILE
+                    variable: EKS_IAM_INSTANCE_PROFILE
```

Comment:
I think you should add DOCKER_USERNAME and DOCKER_PASSWORK here to provide parameters in macros.yaml

Reply:
Updated all the jobs to user DOCKER_USERNAME/DOCKER_PASSWORD instead of ANTREA_DOCKER_USERNAME/ANTREA_DOCKER_PWD